tests/renderer/renderer_store.test.ts

```diff
@@ -1217,6 +1221,14 @@ describe("renderer", () => {
   );
 
   test("cells with a fontsize too big for the row height are clipped", () => {
+    MockCanvasRenderingContext2D.prototype.measureText = function (text: string) {
```

Comment:
euh bof. It makes tests non-deterministic because the prototype is modified for all following tests.